sdk/src/beta9/runner/common.py

```diff
@@ -30,18 +31,18 @@
 
 @dataclass
 class Config:
-    container_id: Optional[str]
-    container_hostname: Optional[str]
-    stub_id: Optional[str]
-    stub_type: Optional[str]
-    workers: Optional[int]
-    keep_warm_seconds: Optional[int]
-    timeout: Optional[int]
+    container_id: str
+    container_hostname: str
+    stub_id: str
+    stub_type: str
+    workers: int
+    keep_warm_seconds: int
+    timeout: int
     python_version: str
     handler: str
-    on_start: Optional[str]
-    callback_url: Optional[str]
-    task_id: Optional[str]
```

Comment:
This is to correct the linting issues.

TODO: Need to test endpoints, taskqueues, and serves because dataclasses do care about something being Optional or not.